src/views/PreOrder.vue

```diff
@@ -0,0 +1,124 @@
+            <ion-card-title>{{ $t("Re-allocate pre-orders") }}</ion-card-title>
+          </ion-card-header>
+          <ion-item>
+            <ion-label>{{ $t("Allocation") }}</ion-label>
+            <ion-button fill="outline" color="danger" slot="end">{{ $t("Run reallocation") }}</ion-button>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap"><p>{{ $t("Re-allocation will re-allocate promise dates on all pre-orders based on upcoming inventory from purchase orders. Promise dates that were manually adjusted will be overriden.") }}</p></ion-label>
+          </ion-item>
+        </ion-card>
+
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>{{ $t("Auto releasing") }}</ion-card-title>
+          </ion-card-header>
+          <ion-item>
+            <ion-label>{{ $t("Run daily") }}</ion-label>
+            <ion-checkbox slot="end" />
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Release preorders")}}</ion-label>
+            <ion-button fill="outline">{{ $t("Release") }}</ion-button>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap"><p>{{ $t("Auto releasing pre-orders will find pre-orders that have promise dates that have passed and release them from fulfillment.") }}</p></ion-label>
+          </ion-item>
+        </ion-card>
+
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>{{ $t("Sync") }}</ion-card-title>
+          </ion-card-header>
+          <ion-item>
+            <ion-label>{{ $t("Add pre-order tag in Shopify") }}</ion-label>
+            <ion-checkbox slot="end" />
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Add shipping dates in Shopify") }}</ion-label>
+            <ion-checkbox slot="end" />
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Manual update") }}</ion-label>
+            <ion-button fill="outline">{{ $t("Run sync now") }}</ion-button>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap"><p>{{ $t("Transfer pre-order related information to Shopify as tags and meta fields.") }}</p></ion-label>
+          </ion-item>
+        </ion-card>
+      </main>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import {
+  IonButton,
+  IonCard,
+  IonCardHeader,
+  IonCardTitle,
+  IonCheckbox,
+  IonContent,
+  IonHeader,
+  IonItem,
+  IonLabel,
+  IonPage,
+  IonTitle,
+  IonToggle,
+  IonToolbar
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'PreOrder',
+  components: {
+    IonButton,
+    IonCard,
+    IonCardHeader,
+    IonCardTitle,
+    IonCheckbox,
+    IonContent,
+    IonHeader,
+    IonItem,
+    IonLabel,
+    IonPage,
+    IonTitle,
+    IonToggle,
+    IonToolbar
+  },
+});
+</script>
+
+<style scoped>
+.content {
```

Comment:
I don't think we need a class for this. we can directly tag main component

Reply:
Done sir